### PR TITLE
Update logos, fonts, and colors

### LINKED ISSUE #4395: Update nextflow logos
The following assets should use the updated nextflow logo 

1. [Notification](https://github.com/nextflow-io/nextflow/blob/dfb5bf092066ea5ac81f3c17ed14e1a42a126366/modules/nextflow/src/main/resources/nextflow/mail/notification.html#L141-L145) email 
2. [Execution report](https://github.com/nextflow-io/nextflow/blob/dfb5bf092066ea5ac81f3c17ed14e1a42a126366/modules/nextflow/src/main/resources/nextflow/trace/ReportTemplate.html#L21-L21)
3. [Timeline report](https://github.com/nextflow-io/nextflow/blob/dfb5bf092066ea5ac81f3c17ed14e1a42a126366/modules/nextflow/src/main/resources/nextflow/trace/TimelineTemplate.html#L21-L21)

### PR DESCRIPTION
**Pull request by bentsherman**
Close #4395

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg-bright.png">
-  <source media="(prefers-color-scheme: light)" srcset="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg.png">
-  <img alt="Nextflow Logo" src="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg.png">
+  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/nextflow-io/trademark/blob/master/nextflow-logo-bg-dark.png">
+  <source media="(prefers-color-scheme: light)" srcset="https://github.com/nextflow-io/trademark/blob/master/nextflow-logo-bg-light.png">
+  <img alt="Nextflow Logo" src="https://github.com/nextflow-io/trademark/blob/master/nextflow-logo-bg-light.png">
```
Comment (ewels) [inline README.md:4]: Please update to use only assets in this repo (add in new logo images if needed).

x-ref https://github.com/nextflow-io/nextflow/issues/4395#issuecomment-1760339290

**Review by ewels — CHANGES_REQUESTED** [review]

**Review by ewels — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg-bright.png">
-  <source media="(prefers-color-scheme: light)" srcset="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg.png">
-  <img alt="Nextflow Logo" src="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg.png">
+  <source media="(prefers-color-scheme: dark)" srcset="docs/_static/nextflow-logo-bg-dark.png">
+  <source media="(prefers-color-scheme: light)" srcset="docs/_static/nextflow-logo-bg-light.png">
+  <img alt="Nextflow Logo" src="docs/_static/nextflow-logo-bg-light.png">
 </picture>
 
 *"Dataflow variables are spectacularly expressive in concurrent programming"*
```
Comment (ewels) [inline README.md:7]: Having the citation under here right at the top of the readme throws me a bit, looks a lot like this should be the DOI for Nextflow...

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg-bright.png">
-  <source media="(prefers-color-scheme: light)" srcset="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg.png">
-  <img alt="Nextflow Logo" src="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg.png">
+  <source media="(prefers-color-scheme: dark)" srcset="docs/_static/nextflow-logo-bg-dark.png">
```
Comment (ewels) [inline README.md:2]: Great! Happier having this here, thanks.

The pedant in me would quite like a version of this image to be made with a bunch of transparent whitespace around it, as it feels a little cramped. But low priority and I can't be bothered to do it myself so don't really expect anyone else to either 😅

**Review by ewels — APPROVED** [review]

**Comment by pditommaso** [comment]
What's the status of this? I see, several open comments

File: modules/nextflow/src/main/resources/nextflow/mail/nextflow200x40.png
Comment (pditommaso) [inline modules/nextflow/src/main/resources/nextflow/mail/nextflow200x40.png:1]: @ewels what tool are you using to optimise the image? I can manage to make it smaller than 10kn for a 490x80px image

File: README.md (hunk 2 above)
Comment (bentsherman) [reply README.md:7]: It's not the nextflow citation, it's a quote from a paper about dataflow programming

File: modules/nextflow/src/main/resources/nextflow/mail/nextflow200x40.png
Comment (bentsherman) [reply modules/nextflow/src/main/resources/nextflow/mail/nextflow200x40.png:1]: I used image magick, but the original size is 200x40 not 490x80

Comment (bentsherman) [reply modules/nextflow/src/main/resources/nextflow/mail/nextflow200x40.png:1]: You must be talking about the full size logos under `docs/_static`. I didn't optimize them, but with image magick I can resize them to 490x80 and they are 12-16 KB

Comment (ewels) [reply modules/nextflow/src/main/resources/nextflow/mail/nextflow200x40.png:1]: My method was to check the size it renders as, double it for retina screens. Resize the original image to that using MacOS Preview. Then take that file and drop it into https://tinypng.com/

Tinypng is awesome :) There's tinyjpeg etc as well.

File: README.md (hunk 2 above)
Comment (ewels) [reply README.md:7]: Yeah I know, still a bit confusing 😅 But doesn't matter.

**Comment by ewels** [comment]
> What's the status of this? I see several open comments

LGTM to me to merge. My remaining comments were super minor and don't need to be addressed.

File: modules/nextflow/src/main/resources/nextflow/mail/nextflow200x40.png
Comment (pditommaso) [reply modules/nextflow/src/main/resources/nextflow/mail/nextflow200x40.png:1]: ah! tinypng did the trick!

**Comment by abhi18av** [comment]
Whoa - we're gonna need new stickers! 😉

**Pull request merged**